kafka-connect-s3/docs/configuration_options.rst

```diff
@@ -33,13 +33,20 @@ Connector
   * Importance: low
 
 ``filename.offset.zero.pad.width``
-  Width to zero pad offsets in store's filenames if offsets are too short in order to provide fixed width filenames that can be ordered by simple lexicographic sorting.
+  Width to zero-pad offsets in store's filenames if offsets are too short in order to provide fixed-width filenames that can be ordered by simple lexicographic sorting.
 
   * Type: int
   * Default: 10
   * Valid Values: [0,...]
   * Importance: low
 
+``append.late.data``
+  If true, the partitions will append data from previous (already commited) partitions into the current, open partition. If false, late data intended for closed partitions will be rolled into files marked with the partition indicated by the extracted time, even if this createsmany small files.
```

Comment:
the "already committed" (typo) part is not correct. You might haven't committed any records in previous partitions. Late data is just late. 

Also, not sure I understand "the partitions will append". You mean the connector?

Similar to the first comment, I'd call "closed partitions" "earlier partitions" 

typo: createsmany